seigniorage.tex

```diff
@@ -122,7 +122,7 @@ \subsubsection*{On-Time Finalization}
   p'_{i}(\mu) = p'(\mu)\quad\text{and}\quad p_{i}(\mu)
 \end{equation}
 
-and introduce for a round $i$ the summit achievable only by in-round messages as $\mathcal{S}_i=((C_j)_{j\in\mathbb{N}}, \sigma,p,p'_{i},q_{OTF})$.
+and introduce for a round $i$ the maximal summit achievable only by in-round messages, $\mathcal{S}_i=((C_j)_{j\in\mathbb{N}}, \sigma,p,p'_{i},q_{OTF})$.
```

Comment:
@afck: w(C_1) here probably refers to the level-1 summit with the maximal quorum?
Maybe the definition above should then be changed, too. So S_i wouldn't be defined using q_OTF, but would be the summit with the maximal q so that it is still level 1. OTF would then mean q ≥ q_OTF?
And for eventual finality, instead of w(C_1) you'd use q.
@osolmaz: Actually, it referred to the level-1 summit with q_OTF. The last thing I remember, we were saying it would be equal to ~`51%*w(V)`. Can you remind me what "maximal quorum" means? Didn't we fix q when reasoning about summit sizes?
@afck: Well, you can either search for the maximal-height summit with a fixed quorum, or for the maximal-quorum summit with a given height.
It sounded like in this case we want the latter.

Reply:
Yes, that's correct. I was confused by the terminology. We simply want for `C_1`, the set of validators who vote for the `PROP` and see each other voting for the `PROP` to have a total weight larger than `q_OTF`.

> OTF would then mean q ≥ q_OTF?

Yes.

> And for eventual finality, instead of w(C_1) you'd use q.

Yes.